pkg/api/render.go

```diff
@@ -39,24 +40,28 @@ func (app *App) Render(fetched *FetchResult) ([]runtime.Object, error) {
 			continue
 		}
 
-		if strings.TrimSpace(data) == "" {
-			log.WithFields(log.Fields{
-				"Name": name,
-			}).Debug("Ignoring empty file.")
-			continue
-		}
+		splitted := regexp.MustCompile("[\n\r]---").Split(data, -1)
```

Comment:
The specs look like there is stuff behind `---` allowed. But I am not entirely sure. The specs are very big.